net/nmap/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem 1.0
 
 name		nmap
 version		7.94
+revision	1
```

Comment:
There is no change on 64-bit except for a compiler warning going away. I could propose incrementing the revision only on macOS versions for 32-bit processors, but I have not done so to avoid confusion.